cgmanifest.json

```diff
@@ -2190,6 +2190,16 @@
         }
       }
     },
+    {
+      "component": {
+        "type": "other",
+        "other": {
+          "name": "libacvp",
+          "version": "1.2.0",
+          "downloadUrl": "https://github.com/cisco/libacvp/archive/v1.2.0.tar.gz"
```

Comment:
> "downloadUrl": "https://github.com/cisco/libacvp/archive/v1.2.0.tar.gz" [](start = 9, length = 72)

I've noticed that the [manifest check](https://github.com/microsoft/CBL-Mariner/pull/607/checks?check_run_id=1863788961) is also acting up for some reason. Could you take a look?

Reply:
Manifest check expects the downloadUrl to be:
`https://github.com/cisco/libacvp/archive/libacvp-1.2.0.tar.gz`

However, the URL I have is correct. Several other packages have this issue (ex: rapidjson), so I won't change this.

Reply:
https://github.com/microsoft/CBL-Mariner/pull/616

This PR should fix that- the manifest check just needs a small change to handle the escaped name/version macros in your `#Source0` line.

Reply:
Loving all of this.:)

---
In reply to: [573258286](https://github.com/microsoft/CBL-Mariner/pull/607#discussion_r573258286) [](ancestors = 573258286)